README.rst

```diff
@@ -173,12 +173,12 @@ Developers". Feel free to cite our work in your research using the following Bib
                       Quinn, Jamie and
                       Wessel, Paul},
       title        = {{PyGMT: A Python interface for the Generic Mapping Tools}},
```

Comment:
L159 should be `@software{pygmt_2022_6349217,`

Reply:
Done